packages/backend/src/tenants/settings/service.test.ts

```diff
@@ -276,21 +270,4 @@ describe('TenantSetting Service', (): void => {
       expect(dbTenantData.filter((x) => !x.deletedAt)).toHaveLength(0)
     })
   })
-
-  describe('pagination', (): void => {
```

Comment:
did you mean to delete?

Reply:
Yes, because it doesn't make sense at this moment, because you have only few keys that you can add ... and we are also doing upsert ... so ... not a lot of data there :)

Reply:
I think we should keep the test because the `getTenantSettingPageForTenant` service method is still used by the tenant settings graphql resolver

Reply:
oh, maybe we can remove that resolver then :D